var/spack/repos/builtin/packages/gearshifft/package.py

```diff
@@ -0,0 +1,74 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Gearshifft(CMakePackage):
+    """Benchmark Suite for Heterogenuous FFT Implementations"""
+
+    homepage = "https://github.com/mpicbg-scicomp/gearshifft"
+    url      = "https://github.com/mpicbg-scicomp/gearshifft/archive/v0.2.0.tar.gz"
+
+    version('0.2.1-lw', 'c3208b767b24255b488a83e5d9e517ea')
+
+    variant('cufft', default=True,
+            description='Compile gearshifft_cufft')
+    #variant('clfft', default=True,
+    #        description='Compile gearshifft_clfft')
+    variant('fftw', default=True,
+            description='Compile gearshifft_fftw')
+    variant('openmp', default=True,
+            description='use OpenMP parallel fftw libraries')
+    # variant('hcfft', default=True,
+    #         description='Not implemented yet')
+
+    # depends_on C++14 compiler, e.g. GCC 5.0+
```

Comment:
You could say:
```python
conflicts('%gcc@:4')
```
but you'd have to do it for every compiler. Maybe we can find a way to express this requirement when we rework compiler support.